infra/blueprint-test/pkg/krmt/krm.go

```diff
@@ -64,6 +64,14 @@ func WithBuildDir(buildDir string) krmtOption {
 	}
 }
 
+func (f *KRMBlueprintTest) GetBuildDir() string {
```

Comment:
nit: lets also move this next to other methods.
```suggestion
func (b *KRMBlueprintTest) GetBuildDir() string {
```